alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+            $ref: '#/definitions/Error'
+        default:
+          $ref: '#/responses/Error'
+  /v1/instances/{id}:
+    parameters:
+      - in: path
+        name: id
+        description: Instance ID
+        type: string
+        required: true
+    get:
+      tags: [alarmInstances]
+      summary: Get an alarm instance
+      description: Gets an alarm instance by its *instanceId*
+      responses:
+        200:
+          $ref: '#/definitions/AlarmInstance'
+    delete:
+      tags: [alarmInstances]
+      summary: Delete an alarm instance
+      description: Deletes an alarm instance by its *instanceId*
+      responses:
+        204:
+          description: No Content
+        401:
+          $ref: '#/responses/Unauthorized'        
+        default:
+          $ref: '#/responses/Error'
+  /v1/instances/{id}/notes:
+    parameters:
+      - in: path
+        name: id
+        description: Instance ID
+        type: string
+        required: true
+    post:
+      tags: [alarmInstances]
+      summary: Add notes to an alarm instance
+      description: Adds one or more notes to an alarm instance.
+      operationId: AddNotesToInstance
+      parameters:
+        - in: body
+          name: Request body
+          required: true
+          schema:
+            type: object
+            required:
+              - notes
+            properties:
+              notes:
+                type: array
+                items:
+                  $ref: '#/definitions/AlarmNote'
+      responses:
+        204:
+          description: No Content
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/query-instances:
+    post:
+      tags: [alarmInstances]
+      summary: Query alarm instances
+      description: Queries for alarm instances
+      operationId: QueryInstances
+      parameters:
+        - in: body
+          name: Request body
+          required: false
+          schema:
+            $ref: '#/definitions/QueryRequest'
+      responses:
+        200:
+          $ref: '#/responses/QueryResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/delete-instances-by-instance-id:
+    post:
+      tags: [alarmInstances]
+      summary: Delete alarm instances
+      description: Deletes multiple alarm instances by their *instanceIds*.
+      operationId: DeleteInstancesByInstanceId
+      parameters:
+        - in: body
+          name: Request body
+          required: true
+          schema:
+            type: object
+            title: Instance IDs to delete
+            properties:
+              instanceIds:
+                description: Instance IDs to delete
+                type: array
+                items:
+                  type: string
+                example:
+                  - 5c33c212e0d6444320d9a9f4
+                  - 5c33c212e0d6444320d9a9f5
+      responses:
+        401:
```

Comment:
This route is missing any success responses.

Reply:
Added a 204